### ISSUE TITLE
Continuation of development?

### ISSUE THREAD

**Issue by FlorianisonGitHub** [Feodor2/Mypal68#540]
What happen to Mypal 68 development? The certain sites are now going out of date.

**Comment by TiredOfWindows** [comment]
Now that you have MyPal 68.14.4 working well for SSE, maybe you're bored and need something else to challenge you? I have several computers that have Windows XP that your browser won't work on because they have PII or Celeron processors with MMX only. Can you make your browser work on them?

**Comment by Feodor2** [comment]
Of course I know, I planned to complete the new version by the end of the year. But I shall make it early while I can, because it may be the total blackout for me before. At least you get fixed bugs from previous versions, most important free memory upon closing a tab, both for single and multi process.

And I do not going to bother with older PII or Celeron, no time for an over-indulgence your knnow.

**Comment by Yggdrax** [comment]
> What about native Windows 2000 support?

As much as I'd like this to have native 2000 support as well, I think it'll be better if the development is more focused into updating the browser to better support newer CSS and javascript standards, at least for now, besides, mypal68 runs perfectly fine on 2000 with KernelEx

**Comment by Feodor2** [comment]
> better support newer CSS and javascript standards

Yes this is a primary goal.

Pleas try help about 2000 there #496.

**Comment by Half-Modern** [comment]
Bored of senseless older OS support. A PC that cannot run **Windows XP Service Pack 3** cannot have enough power to browse the web.

**Comment by Yggdrax** [comment]
> Bored of senseless older OS support. A PC that cannot run **Windows XP Service Pack 3** cannot have enough power to browse the web.

I'll have to disagree about that, there's plenty of hardware that could comfortably run XP SP3, and yet its users choose to run 2000 on it instead, the same logic applies to machines that could run 7 but end up with XP, or even modern PCs running the OS in a VM. The reason for doing it isn't really important, it's like asking why people try to keep using other old OSes like XP, despite it too being extremely old, unsafe, and its period-accurate hardware being extremely weak by today standards.

Besides, if we look at it from the perspective of "senseless older OS support", then, there's no point in keeping around XP/Vista/7/8 software support either, since all of those OSes already reached EOL.

**Comment by MarkusProkott** [comment]
> Bored of senseless older OS support. A PC that cannot run **Windows XP Service Pack 3** cannot have enough power to browse the web.

That's exactly the typical argument for not having XP support at all (or Vista/7/8/9). (And, BTW, I'm also kinda bored of it. ;-)) And it's wrong in itself since «older OS support» makes a lot of different, well-known senses. Just not for certain business models.

**Comment by Half-Modern** [comment]
You're completely missing the point. There are machines that can't run Windows XP Service Pack 3 and people wants to run browsers on them while they are insufficent for that job. There's no point of running Windows 2000 on XP hardware either since 2000 has no advantage over XP. If you "prefer" to run 2000 on XP (or newer)-era machine, then you'll have to accept the consequences.
Also XP-SP3 hardware is pretty weak for web browsing. Athlon 64 x2 & 2 GB RAM should be the lowest spec in order to browse the web nowadays. (In Intel side, this translates to early Core 2 Duo because Pentium D sucked) If you are a masochist, you can use a Pentium 4 to browse the web -unless it's your only option-.

**Comment by NS-Clone** [comment]
> applies to machines that could run 7 but end up with XP,

does w2000 can run XP drivers?

**Comment by NS-Clone** [comment]
@Feodor2 а ты ничего не делол с загрузкой сессий в последней версии?
есть смысол пытатсо обновлять браузера с 1.5К вкладок?

**Comment by Yggdrax** [comment]
> You're completely missing the point. There are machines that can't run Windows XP Service Pack 3 and people wants to run browsers on them while they are insufficent for that job. There's no point of running Windows 2000 on XP hardware either since 2000 has no advantage over XP. If you "prefer" to run 2000 on XP (or newer)-era machine, then you'll have to accept the consequences. Also XP-SP3 hardware is pretty weak for web browsing. Athlon 64 x2 & 2 GB RAM should be the lowest spec in order to browse the web nowadays. (In Intel side, this translates to early Core 2 Duo because Pentium D sucked) If you are a masochist, you can use a Pentium 4 to browse the web -unless it's your only option-.

I mean, by that same logic, there's no point in running XP either, or even 7, since there's no real advantage from running Windows 10 instead... setting that aside, I understand where you are coming from, _it can be_ unreasonable to want modern features running in such old hardware, however, that doesn't mean that software is bound by the same limitations.

As far as this project (and other similar projects) is concerned, it'll always be up to Feodor/the author/the contributors to decide whether or not add support for older systems. I personally don't mind this remaining as a XP-focused project.

> > applies to machines that could run 7 but end up with XP,
> 
> yep but everytime people ask about pentium 3 and nt4.0...
> 
> also does w2000 can run XP drivers?

It depends on the driver, for some you can just force compatibility and it'll run just fine.

**Comment by Half-Modern** [comment]
There are **valid** points in using Windows XP and 7 as opposed to 2000 and 10/11:
XP: Runs better on IDE HDD (than Vista and newer), better software support than and many enhancements over 2000
7: Runs better on SATA HDD (than 10 and newer), better software support and stability than XP/Vista, no forced updates, no long boot times
Personally I'm not one of those who run very old operating systems on overpowered machines, I don't need to install any compatibility patches in order to run XP, my machines are THAT old. That being said, I'd just throw 10 LTSB/LTSC on every single PC if I could afford SSDs for all of them. 
XP - IDE/very slow HDD, single/dual core, <4 GB RAM
7 - SATA HDD, dual core, 2> GB RAM
10 - SSD (worst SSD > best HDD for it), dual/quad core, 4> GB RAM

**Comment by JosefReisinger** [comment]
The discussion of "witch OS should be cancelled" is not really useful, nobody should underestimate PAWLOW with his dog, u remember? If a OS is linked to positive emotions or circumstances in your life, u will never want to give it up, isn't it?  Above all, if this circumstances afterward goes worse, u will stick to the older times more tight ... if it would be different, nobody would mention mypal68 or make talks around it, everybody would prefer Microsoft W11 with Edge ode Ubuntu 22.04 with Firefox in my opinion. So we should be quite open-minded to all of possible Conditionings - W3.11, NT4, W95, W2K, WXP, W7 ... but we all know - the older they are, the huger amount it will take to solve  compat issues ;-)
Greetings from Austria, Joe

**Comment by Win2000DevCommunity** [comment]
> > applies to machines that could run 7 but end up with XP,
> 
> yep but everytime people ask about pentium 3 and nt4.0...
> 
> also does w2000 can run XP drivers?

yes in almost case , also with ACPI 2.0 we get it work on ryzen 7

**Comment by Feodor2** [comment]
I plan to complete next version by the next month and try fix the Culdronshitare

**Comment by JosefReisinger** [comment]
Hey, I had success with latest version mypal 74.1 on top of exkernel win2000, thank you very much for this version! It works smoothly with most websites without crashes, although right menu with all the options is flickering (but no problem with menu bar!); stability of the webpages could be done with psexec (only one of 8 cores, otherwise GAHHH).
Best thankful greetings from DE-AT, Joe

**Comment by unclecaptain5426** [comment]
I had success with latest version mypal 74.1.x on Windows XP SP1 without One-Core API (by just using roytam1's kernelxp wrappers) without crashes.
But for Windows XP RTM without One-Core API (while it works by using roytam1's kernel wrappers), it crashes or freezes and throw out "mypal.exe has encountered a problem and needs to close. We are sorry for the inconvenience." after a few minutes of usage.

**Comment by NS-Clone** [comment]
wut is roytam kernel wrappers? 
he make something? o_O
